test/cases/azure.sh

```diff
@@ -238,7 +239,14 @@ sudo "${CONTAINER_RUNTIME}" run \
     -e ARM_TENANT_ID="${AZURE_TENANT_ID}" \
     -v "${TEMPDIR}":/tmp:Z \
     "${CONTAINER_CLOUD_IMAGE_VAL}" \
-    python cloud-image-val.py "${CIV_OPTIONS[@]}" && RESULTS=1 || RESULTS=0
+    python cloud-image-val.py \
+    ${CIV_OPTION_R:+"$CIV_OPTION_R"} \
+    ${CIV_OPTION_O:+"$CIV_OPTION_O"} \
+    ${CIV_OPTION_T:+"$CIV_OPTION_T"} \
+    ${CIV_OPTION_M:+"$CIV_OPTION_M"} \
+    ${CIV_OPTION_D:+"$CIV_OPTION_D"} \
+    ${CIV_OPTION_S:+"$CIV_OPTION_S"} \
```

Comment:
`CIV_OPTION_T` and `CIV_OPTION_S` don't seem to be set anywhere. Is this intentional?